src/controllers/__tests__/item.test.js

```diff
@@ -110,38 +101,39 @@ const mockUser = {
 };
 
 jest.setTimeout(30000);
-beforeAll(async () => {
-  await clearDatabase();
-  /* 
-    authCookie is needed to authenticate requests made to post, put, and delete items endpoints,
-    without it, all the mentioned requests will not be able to pass the user-authentication middleware.
-    And the reason for assigning all true items owners for the authenticated user is to make sure they
-    pass the item-authorization middleware.
-  */
-  const signUpResponse = await request(server)
-    .post('/api/auth/signup')
-    .set('Content-Type', 'application/json')
-    .send(mockUser);
-  ownerId = signUpResponse.body._id;
-  trueItem.owner = ownerId;
-  trueItem2.owner = ownerId;
-  trueItem3.owner = ownerId;
-  trueItem4.owner = ownerId;
-  const signInResponse = await request(server)
-    .post(`/api/auth/signin`)
-    .set('Content-Type', 'application/json')
-    .send({ username: mockUser.username, password: mockUser.password });
-
-  [authCookie] = signInResponse.headers['set-cookie'];
-});
-
-afterAll(async () => {
-  await clearDatabase();
-  await closeDatabase();
-  server.close();
-});
 
 describe('Items Endpoints', () => {
+  beforeAll(async () => {
```

Comment:
Why is this beforeAll block moved back inside describe? In the last PR I remember it was moved outside.

Reply:
I just merged main and it automatically moved them inside the describe 
I will edit this right now